main.js

```diff
@@ -14,8 +14,8 @@ const { getSchoolTypeList } = require("./src/db/api");
 app.on("ready", () => {
   // Configure the main window
   let mainWindow = new BrowserWindow({
-    width: 1200,
-    height: 665,
+    width: 1400,
+    height: 865,
```

Comment:
do we need this change? desktop size starts with 1200px

Reply:
Maybe it is not needed in fact. But we have to disscuss what resolution should we use as default. What will be te smallest resolution used next year. For example in my previous workplace they are using 6-7years old laptop with HD. I don't think we should strain ourselves to handle the smallest possible screen.